api/src/main/scala/com/rasterfoundry/granary/endpoints/PredictionEndpoints.scala

```diff
@@ -38,5 +38,18 @@ object PredictionEndpoints {
       .in(query[Option[JobStatus]]("status"))
       .out(jsonBody[List[Prediction]])
 
-  val endpoints = List(idLookup, create, list)
+  val addResults =
```

Comment:
The way this is set up, it's not really incumbent on a user to have their job automatically update the endpoint -- they could use the id they get back from creating the prediction if they wanted to and POST the update on their own if their job had retries, for example. I think this wound up being pretty flexible.